src/EditorFeatures/Core/Implementation/Peek/PeekableItemSourceProvider.cs

```diff
@@ -24,8 +24,8 @@ internal sealed class PeekableItemSourceProvider : IPeekableItemSourceProvider
 
         [ImportingConstructor]
         public PeekableItemSourceProvider(
-            IPeekableItemFactory peekableItemFactory,
-            IPeekResultFactory peekResultFactory,
+            [Import(AllowDefault = true)] IPeekableItemFactory peekableItemFactory,
```

Comment:
📝 This is a required export. The implementation is defined and exported in the same assembly where the interface is defined.